bezier_library/src/bezier_grinding_surfacing.cpp

```diff
@@ -568,11 +568,7 @@ bool BezierGrindingSurfacing::generateRobotPosesAlongStripper(const vtkSmartPoin
   }
   free(indices);
 
-  if(!filterNeighborPosesTooClose(point_normal_table, 5e-3))
-  {
-    ROS_ERROR_STREAM("BezierGrindingSurfacing::generateRobotPosesAlongStripper: Cannot filter grinding trajectory");
-    return false;
-  }
+  filterNeighborPosesTooClose(point_normal_table, 5e-3);
```

Comment:
To my mind, these function should keep a boolean return.
By this way, you will see if the `trajectory` will be empty or not in this function.